src/__tests__/Countries.test.js

```diff
@@ -0,0 +1,98 @@
+import { render, screen } from '@testing-library/react';
```

Comment:
- [ ] According to the project requirement your App should have unit tests for several components of the app and some pure functions in Redux code. kindly consider adding more unit tests for several components of the app and some pure functions as requested by the previous reviouwer 💯 ✍🏻